Sources/Common/Core/Points/index.js

```diff
@@ -89,10 +89,10 @@ const DEFAULT_VALUES = {
 
 // ----------------------------------------------------------------------------
 
-export function extend(publicAPI, model, initialValues = {}) {
-  Object.assign(model, DEFAULT_VALUES, initialValues);
+export function extend(publicAPI, model) {
```

Comment:
for good practice I will keep initialValues arg. Maybe we just need something to mention that we don't use it...

Reply:
I do not understand why we do not want to support initialValues